<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qSlicerSettingsGeneralPanel</class>
 <widget class="ctkSettingsPanel" name="qSlicerSettingsGeneralPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>415</width>
    <height>306</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>General</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="DefaultScenePathLabel">
     <property name="toolTip">
      <string>Directory where scenes are saved to by default</string>
     </property>
     <property name="text">
      <string>Default scene location:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="ctkDirectoryButton" name="DefaultScenePathButton"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="ShowSplashScreenLabel">
     <property name="text">
      <string>Disable splash screen:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QCheckBox" name="ShowSplashScreenCheckBox">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="ctkLanguageComboBox" name="LanguageComboBox"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="LanguageLabel">
     <property name="text">
      <string>Language</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="ConfirmRestartLabel">
     <property name="text">
      <string>Confirm on restart:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QCheckBox" name="ConfirmRestartCheckBox">
     <property name="text">
      <string/>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="ConfirmExitLabel">
     <property name="text">
      <string>Confirm on exit:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QCheckBox" name="ConfirmExitCheckBox">
     <property name="text">
      <string/>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="ConfirmSceneCloseLabel">
     <property name="text">
      <string>Confirm on scene close:</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QCheckBox" name="ConfirmSceneCloseCheckBox">
     <property name="text">
      <string/>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="DocumentationBaseURLLabel">
     <property name="text">
      <string>Documentation base URL:</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QLineEdit" name="DocumentationBaseURLLineEdit">
     <property name="toolTip">
      <string>Specify documentation location.
Available placeholders: {language}, {version}.
Default: https://slicer.readthedocs.io/{language}/{version}</string>
     </property>
    </widget>
   </item>
   <item row="9" column="1">
    <widget class="QSpinBox" name="NumOfRecentlyLoadedFiles">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>50</number>
     </property>
     <property name="value">
      <number>10</number>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QLabel" name="NumOfRecentlyLoadedFilesLabel">
     <property name="text">
      <string>Max. number of 'Recently Loaded' menu items:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="SlicerRCFileLabel">
     <property name="toolTip">
      <string>Python script that is executed after the application is started</string>
     </property>
     <property name="text">
      <string>Application startup script:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="SlicerRCFileValueLabel">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="frame">
        <bool>true</bool>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="SlicerRCFileOpenButton">
       <property name="toolTip">
        <string>Open Slicer resource file</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="DocumentationBaseURLLabel_2">
     <property name="text">
      <string>Module documentation URL:</string>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <widget class="QLineEdit" name="ModuleDocumentationURLLineEdit">
     <property name="toolTip">
      <string>Specify URL for module documentation.
Available placeholders: {documentationbaseurl}, {lowercasemodulename}.
Default: {documentationbaseurl}/user_guide/modules/{lowercasemodulename}.html</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ctkDirectoryButton</class>
   <extends>QWidget</extends>
   <header>ctkDirectoryButton.h</header>
  </customwidget>
  <customwidget>
   <class>ctkSettingsPanel</class>
   <extends>QWidget</extends>
   <header>ctkSettingsPanel.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ctkLanguageComboBox</class>
   <extends>QComboBox</extends>
   <header>ctkLanguageComboBox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
